svc/loginservice/cmd/loginservice/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	"github.com/ampazdev/simple-go-project/svc/loginservice/constant"
+	"github.com/ampazdev/simple-go-project/svc/loginservice/internal/app"
+	"github.com/ampazdev/simple-go-project/svc/loginservice/internal/delivery/rest/echo"
+)
+
+func main() {
+	env := os.Getenv(constant.OsEnvDefaultEnvironment)
+	if env == "" {
+		env = constant.OsEnvDefaultEnvironment
+	}
+
+	//configPath := flag.String("config", "", "specifies configuration's file path")
+	flag.Parse()
+
+	log.SetFlags(log.Ldate | log.Ltime | log.Lmicroseconds | log.Llongfile)
+
+	prodServiceApp, err := app.Newloginservice(constant.ConfigProjectFilepath)
```

Comment:
I think this is not a product service right? wkwkwk :smile:

Reply:
okaay